generators/app/templates/scripts/_rollup.config.js

```diff
@@ -78,10 +59,52 @@ const externals = {
   ])
 };
 
+/* configuration for plugins */
+const primedPlugins = {
+  babel: babel({
+    babelrc: false,
+    exclude: 'node_modules/**',
+    presets: [
+      ['env', {loose: true, modules: false, targets: {browsers: pkg.browserslist}}]
+    ],
+    plugins: [
+      'external-helpers',
+      'transform-object-assign'
+    ]
+  }),
+  commonjs: commonjs({sourceMap: false}),
+  json: json(),
+  multiEntry: multiEntry({exports: false}),
+  resolve: resolve({browser: true, main: true, jsnext: true}),
+  uglify: uglify({output: {comments: 'some'}}, minify),
+  /* we have to shim externals to a string to work-around amd bugs in rollup */
+  shim: shim(externals.umd.reduce(function(newObj, name) {
+    // set the shim to the global or empty object
+    newObj[name] = globals.umd[name] || '{}';
+
+    // if the shim is not window, document, or empty object add window to the front of it.
+    if (newObj[name] !== 'window' && newObj[name] !== 'document' && newObj[name] !== '{}') {
+      newObj[name] = 'window.' + newObj[name];
```

Comment:
we should probably do a check along the lines of
```
typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {}
```
to get the global object to be defensive.

Reply:
done